lxd/storage/drivers/driver_ceph.go

```diff
@@ -303,6 +342,13 @@ func (d *ceph) Validate(config map[string]string) error {
 		//  defaultdesc: `32`
 		//  shortdesc: Number of placement groups for the OSD storage pool
 		"ceph.osd.pg_num": validate.IsAny,
+		// lxdmeta:generate(entities=storage-ceph; group=pool-conf; key=ceph.osd.pool_size)
+		//
+		// ---
+		//  type: string
+		//  defaultdesc: `3`
+		//  shortdesc: Number of RADOS object replicas. Set to 1 for no replication.
+		"ceph.osd.pool_size": validate.Optional(validate.IsInRange(1, math.MaxInt64)),
```

Comment:
While technically correct, maybe we could put an upper bound that's a little less... gigantic :) Hardcoding `255` feels future-proof enough I'd say.

Reply:
I just noticed that `ceph.osd.pg_num` has no such validation.

Reply:
Set it to 255 now. Interestingly Ceph totally lets you use pg_num=0 without erroring out. Not sure what it does though.